src/ui/Input/index.ts

```diff
@@ -1,11 +1,7 @@
 import DefaultInput from './DefaultInput/DefaultInput';
+import OTPInput from './OTPInput/OTPInput';
 import PasswordInput from './PasswordInput/PasswordInput';
 import SearchInput from './SearchInput/SearchInput';
-import OTPInput from './OTPInput/OTPInput';
+import InputRange from './inputRange/InputRange';
```

Comment:
```suggestion
import RangeInput from './rangeInput/RangeInput';
```

Замени также везде названия